app/assets/stylesheets/publication.scss

```diff
@@ -1,6 +1,6 @@
 #modal {
   .publication {
-    padding: 0px;
+    padding: 0;
```

Comment:
I noticed you changed this from `0px` to `0` 

I am just wondering why for my own education.

Reply:
@knowuh That's a code style habit I picked up a long time ago. I forget where exactly, but I read it's best to leave off units when a value is zero in CSS. Since you could be using any one of several units, it could be zero _anythings_, so it doesn't make sense to specify a particular unit type. It's not necessary and doesn't result in anything looking or behaving differently, but I think the idea makes good sense and it helps minimize the code.